content/blog/colour-scales.md

```diff
@@ -0,0 +1,72 @@
+---
+author: "Emma Russell"
+date: 2020-05-30
+title: "Shades of meaning: visualising geographical data with colour in a data-driven web application"
+best: false
+tags:
+ - visualisation
+ - UI
+---
+
+Colour and maps are two crucial aspects of data visualisation, and putting them together is a classic technique for revealing geographical patterns in data. 
+
+<img src="/img/cscale_smith_map.jpg" alt="William Smith geological map"/>
+*William Smith's [geological map](https://www.nhm.ac.uk/discover/first-geological-map-of-britain.html) of 1815*
```

Comment:
I needed a newline before this one for it to render the link correctly.

Reply:
I also think it would be nice if the font was a bit smaller, and maybe right aligned for the figure caption, but I couldn't work out how to do it while also making Hugo render the link correctly. Using `<figure><img><caption>`... wouldn't render the link. Surrounding with a `<div style="text-align:right">` gets ignored... I dunno....

Reply:
Going to try doing all these with the figure shortcode as Rich suggests